install.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env zsh
+
+conda activate base
+yes | conda create --name fpst_venv
```

Comment:
Deal with an environment already with that name

Reply:
Yes currently it just overwrites it but we probably don't want to do that...

Reply:
Fixed here: https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox/pull/2/commits/9c0354f190cd7081c8077c8427d9c90457d50bcf
I added a 12-character hexadecimal into the name; I think the probability of the user having a conda env with that name is exceedingly low so it shouldn't conflict. If the name does exist, we are assuming it was used in the install for this plugin, in which case we will reinstall and overwrite, which is the desired behaviour.